web/blockframe.html

```diff
@@ -41,56 +40,56 @@
     <script src="../blockly/blocks/procedures.js"></script>
     <script src="../blockly/blocks/funblocks-procedures.js"></script>
     <script src="../blockly/msg/js/en.js"></script>
+  </head>
 
-</head>
-
-<body>
-    <div id="blocklyDiv" style="height: 100%; width: 100%"></div>
+  <body>
+    <div id="blocklyDiv" style="height: 100%; width: 100%;"></div>
 
     <script src="../js/blocks_runmain.js"></script>
     <script>
-        let upperParent = null;
-        let frameId = null;
-
-        function loadXml(data) {
-            let workspace = Blockly.getMainWorkspace();
-            if (!workspace) {
-                setTimeout(() => {
-                    loadXml(data)
-                }, 100);
-                return; // Try again a little later
-            }
-            let xml = Blockly.Xml.textToDom(data);
-            Blockly.Xml.domToWorkspace(xml, workspace);
-
-            let metrics = workspace.getMetrics();
-            let maxX = metrics.contentWidth + 50;
-            let maxY = metrics.contentHeight + 50;
-            frameId.setAttribute('width', maxX);
-            frameId.setAttribute('height', maxY);
-        }
-
-        function setId(j_id_param) {
-            frameId = j_id_param;
+      let upperParent = null;
+      let frameId = null;
+
+      function loadXml(data) {
+        let workspace = Blockly.getMainWorkspace();
```

Comment:
Hmm, judging by all these `let`s that should be `const`, ESLint seems to be skipping this.  I guess because it's a mixed file type, but that means it's more important to avoid these big blocks of inline JavaScript.

Reply:
Probably. Indeed, inline JS should be avoided whenever possible.

Reply:
Okay, I'll add that to the code health github project.